src/api/spec/models/event_subscription/find_for_event_spec.rb

```diff
@@ -111,6 +158,28 @@
         let!(:maintainer) { create(:group) }
       end
 
+      context 'with a user who watching the project' do
+        let!(:watcher) { create(:confirmed_user) }
+        let!(:project) { create(:project) }
+        let!(:comment) { create(:comment_project, commentable: project) }
+
+        let!(:default_subscription) do
+          create(:event_subscription_comment_for_project, receiver_role: 'watcher', user: nil, group: nil)
+        end
+
+        before do
+          watcher.add_watched_project(project.name)
+        end
+
+        it 'returns a new subscription for the watcher based on the default subscription' do
+          result_subscription = subject.find { |subscription| subscription.subscriber == watcher }
+
+          expect(result_subscription.id).to be_nil
+          expect(result_subscription.eventtype).to eq(default_subscription.eventtype)
+          expect(result_subscription.channel).to eq(default_subscription.channel)
+        end
+      end
```

Comment:
That test looks pretty similar to the one above (line 118). I assume the way the default_subscription is set up makes the difference. But it's hard to spot the difference between the two tests, like what effect does the [channel: :instant_email](https://github.com/openSUSE/open-build-service/pull/3992/files#diff-ad9968a7b553554b3b3d013f2c2bc76eR114 ) have? That doesn't seem to be reflected in the test.

IMO it would be worth to move this to a shared example to reduce the duplication:-)